sites/skeleton.dev/src/examples/components/tags-input/react/default.tsx

```diff
@@ -0,0 +1,27 @@
+import { TagsInput } from '@skeletonlabs/skeleton-react';
```

Comment:
As always, I recommend viewing the v3 docs for guidance on various examples to showcase:

https://www.skeleton.dev/docs/components/tags-input/svelte